helper/helper-common/src/main/java/com/linkedin/avroutil1/compatibility/Jackson1Utils.java

```diff
@@ -11,13 +11,22 @@
 import java.util.Map;
 import org.apache.avro.AvroRuntimeException;
 import org.apache.avro.generic.GenericData;
+import org.codehaus.jackson.JsonFactory;
 import org.codehaus.jackson.JsonNode;
+import org.codehaus.jackson.JsonParser;
 import org.codehaus.jackson.map.ObjectMapper;
 import org.codehaus.jackson.node.JsonNodeFactory;
+import org.codehaus.jackson.node.TextNode;
 
 
 public class Jackson1Utils {
-  static final String BYTES_CHARSET = "ISO-8859-1";
+  private static final String BYTES_CHARSET = "ISO-8859-1";
+  private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
```

Comment:
do we want to allow some of the non standard stuff? jackson as used by avro does :-(

Reply:
Sure, happy to add, but I don't know what they are. Could you give some pointers?

Reply:
Radai pointed out offline that Avro's Schema class uses a Jackson mapper configured to allow comments. I think for now, it's okay not to support comments in prop getter/setter methods. We can revisit if somebody runs into an issue.